EvaluateReversePolishNotation.java

```diff
@@ -0,0 +1,62 @@
+package CompetitiveCoding_Dec27;
+
+import java.util.HashSet;
+import java.util.Set;
+import java.util.Stack;
+
+class EvaluateReversePolishNotation {
+	
+    public int evalRPN(String[] tokens) {
+        
+        /*
+        Time complexity: O(n) where n is length of tokens string
+        Space complexity: O(1) because though an additional stack and set is used in the solution, stack will have only 2 elements at any point of time. Set is of fixed size. i.e.4 for each of the operators.
```

Comment:
Space: O(n), The stack can have more than 2 values at any point of time, think about cases where the Operators are all at the end. EX: ["4", "13", "5", "/", "+"]